src/mds/MDSAuthCaps.h

```diff
@@ -93,6 +93,17 @@ struct MDSCapSpec {
   bool allow_full() const {
     return (caps & FULL);
   }
+
+  unsigned get_caps() {
+    return caps;
+  }
+
+  void set_caps(unsigned _caps) {
```

Comment:
UUIC, `unsigned` alone is discouraged in new code but `MDSCapSpec::caps` is already typed that way.

Reply:
Done.